File: src/test/java/com/ject/studytrip/auth/fixture/KakaoLoginRequestFixture.java
package com.ject.studytrip.auth.fixture;

import com.ject.studytrip.auth.presentation.dto.request.KakaoLoginRequest;

public class KakaoLoginRequestFixture {
    private String code = "valid-code";

    public KakaoLoginRequestFixture withCode(String code) {
        this.code = code;
        return this;
    }

    public KakaoLoginRequest build() {
        return new KakaoLoginRequest(code);
    }
}

File: src/test/kotlin/com/ject/studytrip/auth/fixture/KakaoLoginRequestFixture.kt
package com.ject.studytrip.auth.fixture

import com.ject.studytrip.auth.presentation.dto.request.KakaoLoginRequest

class KakaoLoginRequestFixture(
    private val code: String = "kakao-auth-code-1234567890",
) {
    fun build(): KakaoLoginRequest = KakaoLoginRequest(code)
}
